spec/javascripts/fixtures/NewsletterSticky.html

```diff
@@ -1,9 +1,11 @@
 <div id="fixture-1">
   <div class="news-signup-sticky news-signup-sticky--positioned" data-dough-component="NewsletterSticky">
     <form onsubmit="return false;">
-      <button class="news-signup-sticky__close">
+      <button class="news-signup-sticky__close" id="first-focusable">
```

Comment:
Only `l-` and `js-` are defined in our styleguide (https://www.moneyadviceservice.org.uk/en/styleguide/css). The `t-` has been heavily used across all the MAS project for test hooks and I've not seen anything that uses them for theme styles.